### ISSUE TITLE
img alt text is not accessibility friendly

### ISSUE THREAD

**Issue by discospiff** [Anukriti007/XMLProjectIS#36]
alt text does not describe the image in adequate detail for someone who cannot see it.

https://github.com/Anukriti007/XMLProjectIS/blob/580a75f7d9cac257eaffd6b7b390aa6a71d929ce/XMLProjectIS/Pages/Index.cshtml#L95
